Source/UI/IdentityUIConfiguration.swift

```diff
@@ -104,19 +125,30 @@ public struct IdentityUIConfiguration {
             theme: theme ?? self.theme,
             isCancelable: isCancelable ?? self.isCancelable,
             isSkippable: isSkippable ?? self.isSkippable,
+            useBiometrics: useBiometrics ?? self.useBiometrics,
             presentationHook: presentationHook ?? self.presentationHook,
             tracker: tracker ?? self.tracker,
             localizationBundle: localizationBundle ?? self.localizationBundle,
             appName: appName ?? self.appName
         )
     }
+
+    /**
+     Call this to enroll biometrics login
+
+    - parameter useBiometrics: If you want to enable biometrics authentication.
+    */
+    public func useBiometrics(_ useBiometrics: Bool) {
```

Comment:
This function is redundant since `useBiometrics` is a public computed property implementing the same behavior. So all usages should be replaced with `configuration.useBiometrics = value`.